Documentation/Upgrade/rook-upgrade.md

```diff
@@ -256,3 +256,14 @@ This cluster is finished:
 At this point, the Rook operator should be running version `rook/ceph:v1.13.0`.
 
 Verify the CephCluster health using the [health verification doc](health-verification.md).
+
+### **6. Disable CSI holder pods**
+CSI "holder" pods are frequently reported objects of confusion and struggle in Rook. Because of
+this, they are being deprecated and will be removed in Rook v1.16.
+
+If there are any CephClusters that use `network.provider: "multus"`, or if the operator config
```

Comment:
The default is to use host networking, so we don't expect many users to be affected, right? Should we add a note that they would only be affected it they changed those default network settings? 
```suggestion
If there are any CephClusters that use the non-default network setting `network.provider: "multus"`, or if the operator config
```

Reply:
Seems like a good note. Added 👍